src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/CaseloadRepository.kt

```diff
@@ -1,8 +1,12 @@
 package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 import org.springframework.stereotype.Repository
 import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Caseload
 
 @Repository
-interface CaseloadRepository : JpaRepository<Caseload, String>
+interface CaseloadRepository : JpaRepository<Caseload, String> {
+  @Query("select distinct c.id from Caseload c join CaseloadCurrentAccountsBase b on c.id = b.id.caseloadId order by c.id asc")
```

Comment:
Just wondering about the query - will it still return caseloads that have a balance of 0 - as there are entries in the table with a value of 0